### PR TITLE
feat(content): Standardize tooltips

### PR DESCRIPTION
**Pull request by tibetiroka**
**Content (Artwork / Missions / Jobs)**

## Summary
This PR aims to standardize tooltips, as listed in Derpy's to-do list. I also moved some tooltips to keep them in alphabetical order within their category, and fixed some minor copy-paste bugs.

Tooltip (groups) that I'm not sure how to tackle:
- acceleration
- sells for
- fuel/leakage
- gun/turret ports
- heat dissipation
- ramscoop
- tactical scan
- the new setting tooltips

The tone and style of tooltips is still very much up for discussion. Currently, these are some of the rough templates I'm trying to follow:

|`<effect>`|`<period>`|`<cause/action>`|
|---------------|---------------|----------------------|
|Energy consumed|per second|when firing this afterburner.|

|`<effect>`|`<component>`|`<action>`|
|--------------|----------------------|--------------|
|The number of people (passengers or crew)|your ship|can hold.|

|`<effect>`|`<attack>`|
|------------|------------|
|Protection provided|against burn damage.|

|`<effect>`|`<period>`|
|------------|-------------|
|Burn damage resisted|per second.|

|`<issue>`|`<fix>`|
|-------------|---------|
|This ship does not have any source of energy to power its systems.|You must install some sort of generator.|

|`<outfitting issue>`|`<explanation>`|
|-------------|-----------|
|This ship has a reverse thruster but no forward thruster.|It could be very difficult to maneuver.|

## Save File
N/A

### CODE REVIEW

**Comment by tibetiroka** [comment]
I'm marking this ready for review as nearly everything is done, and I don't seem to get much feedback on this as a draft.

Also, I'm really not sure what to do with most of the remaining ones.

File: data/_ui/tooltips.txt [hunk 1]
```diff
@@ -87,11 +90,14 @@ tip "capture attack:"
...
+tip "cargo:"
+	`Tons of cargo this ship is carrying right now, out of its maximum.`
+
 tip "cargo scan:"
-	`Lets you scan your target ship's cargo from up to this distance away.`
+	`Allows scanning your target ship's cargo from up to this distance away.`
 
 tip "cargo scan power:"
-	`The range of this cargo scanner is proportional to the square root of this value.`
+	`Allows scanning your target ship's cargo. The range of this cargo scanner is proportional to the square root of this value.`
```
Comment (W1zrad) [inline data/_ui/tooltips.txt:100]:
```suggestion
	`Allows scanning of your target ship's cargo. The range of this cargo scanner is proportional to the square root of this value.`
```

**Review by W1zrad — COMMENTED** [review]
I think this would be good for @Amazinite to look at rather than myself. Everything seems consistent enough, but this is so technical that I feel like the party who wants to see this fixed (Amazinite) will have more specific preferences.

File: data/_ui/tooltips.txt [hunk 2]
```diff
@@ -151,7 +154,7 @@ tip "cost:"
 	`Cost, in credits. If you have sold any used (depreciated) items today, you can buy them back for the same price you sold them for.`
 
 tip "discharge protection:"
-	`Protection against weapons that have a discharge effect. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in discharge damage, while a total value of 11 only results in a 10 percent reduction.`
+	`Protection provided shield discharge damage. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in discharge damage, while a total value of 11 only results in a 10 percent reduction.`
```
Comment (roadrunner56) [inline data/_ui/tooltips.txt:157]:
```suggestion
	`Protection provided against shield discharge damage. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in discharge damage, while a total value of 11 only results in a 10 percent reduction.`
```

File: data/_ui/tooltips.txt [hunk 3]
```diff
@@ -166,10 +169,10 @@ tip "discharge resistance heat:"
 	`Heat generated per second when resisting discharge damage.`
 
 tip "disruption protection:"
-	`Protection against weapons that have a disruption effect. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in disruption damage, while a total value of 11 only results in a 10 percent reduction.`
+	`Protection provided against shield disruption damage. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in disruption damage, while a total value of 11 only results in a 10 percent reduction.`
 
 tip "disruption resistance:"
-	`How much shield disruption damage this ship can resist per second.`
+	`Shield disruption damage resised per second.`
```
Comment (roadrunner56) [inline data/_ui/tooltips.txt:175]:
```suggestion
	`Shield disruption damage resisted per second.`
```

File: data/_ui/tooltips.txt [hunk 4]
```diff
@@ -853,109 +853,109 @@ tip "firing burn / second:"
...
-	`The amount of fuel lost by the target per shot when hit by this weapon. If its shields are up, the loss will be cut in half. A negative amount means fuel is added.`
+	`Fuel lost by the target per shot when hit by this weapon. Only half as effective when the target's shields are up. A negative amount means fuel is added.`
 
 tip "heat damage / shot:"
-	`Each shot increases the target ship's heat by this amount, or half this amount if its shields are up.`
+	`Increases the target ship's heat by this amount per shot. Only half as effective when the target's shields are up.`
 
 tip "energy damage / shot:"
-	`Each shot reduces the target ship's energy by this amount, or half this amount if its shields are up.`
+	`Reduces the target ship's energy by this amount per shot, or half this amount if its shields are up.`
```
Comment (roadrunner56) [inline data/_ui/tooltips.txt:913]:
```suggestion
	`Reduces the target ship's energy by this amount per shot. Only half as effective when the target's shields are up.`
```

**Review by roadrunner56 — CHANGES_REQUESTED** [review]

File: data/_ui/tooltips.txt [hunk 5]
```diff
@@ -553,25 +565,13 @@ tip "slowing resistance heat:"
...
-
-tip "acceleration:"
-	`How quickly this ship gains speed. The higher a ship's mass (including the mass of any cargo or fighters it is carrying), the slower it accelerates.`
-
-tip "movement:"
-	`How quickly this ship can accelerate and turn.`
-
-tip "movement (full - no cargo):"
-	`How quickly this ship can accelerate and turn when its cargo hold is full versus when it is empty.`
+	`Allow viewing your current target's crew complement, fuel, heat, and energy levels when it is within range. The range is proportional to the square root of this value.`
```
Comment (roadrunner56) [inline data/_ui/tooltips.txt:574]:
```suggestion
	`Allows viewing your current target's crew complement, fuel, heat, and energy levels when it is within range. The range is proportional to the square root of this value.`
```

File: data/_ui/tooltips.txt [hunk 6]
```diff
@@ -745,94 +745,94 @@ tip "Special:"
...
-	`Amount of heat damage over time dealt by this weapon per second when firing.`
+	`Heat damage dealt over time by this weapon per second.`
 
 tip "% shield damage / second:"
-	`Relative shield damage, converted to damage per second to allow easy comparisons between different weapons.`
+	`Shield damage dealt per second, as a percentage of the target's total shield strength.`
 
 tip "% hull damage / second:"
-	`Relative hull damage, converted to damage per second to allow easy comparisons between different weapons.`
+	`Hull damage dealt per second, as a percentage of the taget's total hull strength.`
```
Comment (roadrunner56) [inline data/_ui/tooltips.txt:802]:
```suggestion
	`Hull damage dealt per second, as a percentage of the target's total hull strength.`
```

File: data/_ui/tooltips.txt [hunk 7]
```diff
@@ -745,94 +745,94 @@ tip "Special:"
...
 
 tip "% minable damage / second:"
-	`Relative minable damage, converted to damage per second to allow easy comparisons between different weapons.`
+	`Minable damage dealt per second, as a percentage of the target's total healt.`
```
Comment (roadrunner56) [inline data/_ui/tooltips.txt:805]:
```suggestion
	`Minable damage dealt per second, as a percentage of the target's total health.`
```

File: data/_ui/tooltips.txt [hunk 8]
```diff
@@ -745,94 +745,94 @@ tip "Special:"
...
 
 tip "% fuel damage / second:"
-	`The amount of fuel lost as a percentage of a target's total fuel per second when hit by this weapon. If its shields are up, the loss will be cut in half.`
+	`Fuel loss caused per second as a percentage of the the target's total fuel capacity. Only half as effective when the target's shields are up.`
```
Comment (roadrunner56) [inline data/_ui/tooltips.txt:808]:
```suggestion
	`Fuel loss caused per second as a percentage of the target's total fuel capacity. Only half as effective when the target's shields are up.`
```

File: data/_ui/tooltips.txt [hunk 9]
```diff
@@ -853,109 +853,109 @@ tip "firing burn / second:"
...
-	`Each shot increases the target's hull corrosion by this amount. Hull discharge continuously reduces the target's hull and slowly wears off over time.`
+	`Hull damage dealt over time by this weapon per shot.`
 
 tip "leak damage / shot:"
-	`Each shot causes the target to leak by this amount. Leak continuously decreases the target's fuel and slowly wears off over time.`
+	`Fuel damage dealt over time by this weapon per shot.`
 
 tip "burn damage / shot:"
-	`Each shot causes the target to burn by this amount. Burn continuously increases the target's heat and slowly wears off over time.`
+	`Fuel damage dealt over time by this weapon per shot.`
```
Comment (roadrunner56) [inline data/_ui/tooltips.txt:937]: Is this supposed to be the same as the tooltip above?

**Review by roadrunner56 — CHANGES_REQUESTED** [review]

File: data/_ui/tooltips.txt (hunk 7 above)
Comment (tibetiroka) [reply data/_ui/tooltips.txt:805]: ~~Don't we have a spellcheck CI or something?~~

File: data/_ui/tooltips.txt (hunk 9 above)
Comment (tibetiroka) [reply data/_ui/tooltips.txt:937]:
```suggestion
	`Heat damage dealt over time by this weapon per shot.`
```

File: data/_ui/tooltips.txt [hunk 10]
```diff
@@ -304,19 +307,19 @@ tip "illegal:"
...
 
 tip "ion protection:"
-	`Protection against weapons that do ion damage. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in ion damage, while a total value of 11 only results in a 10 percent reduction.`
+	`Protection provided against ion damage. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in ion damage, while a total value of 11 only results in a 10 percent reduction.`
 
 tip "scramble protection:"
-	`Protection against weapons that do scrambling damage. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in ion damage, while a total value of 11 only results in a 10 percent reduction.`
+	`Protection provided against weapons that do scrambling damage. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in scramble damage, while a total value of 11 only results in a 10 percent reduction.`
```
Comment (Amazinite) [inline data/_ui/tooltips.txt:319]: This one didn't get the full protection tooltip treatment.

File: data/_ui/tooltips.txt [hunk 11]
```diff
@@ -343,25 +346,25 @@ tip "scramble resistance heat:"
...
-	`How much fuel this drive consumes, per 100 tons of mass. Additive with jump fuel.`
+	`Fuel consumed per 100 tons of mass when jumping. Additive with jump fuel.`
 
 tip "jump base mass:"
-	`A modifier to the impact your ship's mass has on the jump mass cost. Ships below this mass will have a cheaper jump cost than the base jump fuel cost, while ships above this mass will have more expensive jumps.`
+	`Modifies the impact your ship's mass has on the jump mass cost. Ships below this mass will have a cheaper jump cost than the base jump fuel cost, while ships above this mass will have more expensive jumps.`
 
 tip "leak protection:"
-	`Protection against weapons that cause fuel leakage. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in leak damage, while a total value of 11 only results in a 10 percent reduction.`
+	`Protection provided against fuel leakage. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in leak damage, while a total value of 11 only results in a 10 percent reduction.`
```
Comment (Amazinite) [inline data/_ui/tooltips.txt:364]: Should this be changed to "Protection provided against leak damage"?

File: data/_ui/tooltips.txt [hunk 12]
```diff
@@ -373,28 +376,37 @@ tip "leak resistance heat:"
...
+	`The number of star systems included in this map.`
 
 tip "mass with no cargo:"
 	`Mass of this ship when not carrying any cargo or fighters. Cargo increases mass, which reduces acceleration and turn rate and increases maximum heat before overheating.`
 
 tip "mass:"
 	`Mass, in tons. A ship's mass determines how fast it turns and accelerates and its maximum heat before overheating.`
 
+tip "max speed:"
+	`The maximum speed of the ship when using thrusters but no afterburners.`
```
Comment (Amazinite) [inline data/_ui/tooltips.txt:391]: Max speed actually can display the maximum speed while thrusters are active if there are no normal thrusters installed.

Also spotted that this says "the ship" while other tooltips around it say "this ship." Might be worth running through all the tooltips to standardize that. For these tooltips that only appear in the ship info sidebar, I think "this ship" makes sense.
```suggestion
	`The maximum speed of this ship when using thrusters. Excludes the effect of afterburners, unless there are no normal thrusters installed.`
```

File: data/_ui/tooltips.txt [hunk 13]
```diff
@@ -406,22 +418,22 @@ tip "outfit scan opacity:"
...
 tip "overheat damage rate:"
-	`How much hull damage this ship will take per second that it is overheated. Overheat damage scales with how far past the overheat threshold the ship is.`
+	`Hull damage taken per second when overheated. Overheat damage scales with how far past the overheat threshold the ship is.`
 
 tip "overheat damage threshold:"
 	`Increases the percentage past overheated at which point overheat damage begins.`
 
 tip "piercing protection:"
-	`Protection against weapons that pierce shields. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in damage leak, while a total value of 11 only results in a 10 percent reduction.`
+	`Protection provided against shield piercing damage. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in damage leak, while a total value of 11 only results in a 10 percent reduction.`
```
Comment (Amazinite) [inline data/_ui/tooltips.txt:436]: Piercing isn't an actual damage type, it's a modifier to shield and hull damage, so I don't think that calling it "shield piercing damage" makes sense. It's also not protection in the same way that the other protection attributes reduce the total amount of damage received, as piercing protection just bounces damage that would've hit the hull back to the shields. We could probably also use a different term than "damage leak," or maybe explain what is meant by that (although that may be redundant with reading piercing's tooltip).

Perhaps look to piercing resistance's tooltip.

File: data/_ui/tooltips.txt [hunk 14]
```diff
@@ -448,10 +460,10 @@ tip "reverse thrusting heat:"
 	`Heat produced per second when firing reverse thrusters.`
 
 tip "reverse thrusting shields:"
-	`Shields displaced, consumed, or destroyed, per second when firing reverse thrusters.`
+	`Shields displaced, consumed, or destroyed per second when firing reverse thrusters.`
 
 tip "reverse thrusting hull:"
-	`Hull that flakes off, breaks apart, or is otherwise compromised, per second when firing reverse thrusters.`
+	`Hull that flakes off, breaks apart, or is otherwise compromised per second when firing reverse thrusters.`
```
Comment (Amazinite) [inline data/_ui/tooltips.txt:466]: I feel like these should just be simplified to "(Shield || Hull) damage taken per second..." instead of providing explanations for why the shields or hull might be getting used up. Such explanations are what the outfit's description should provide.

File: data/_ui/tooltips.txt [hunk 15]
```diff
@@ -853,109 +853,109 @@ tip "firing burn / second:"
...
-	`How well this weapon maintains an infrared target lock, which depends on the target ship's heat level and distance from the missile; IR-guided missiles become more accurate the closer they are to their targets.`
+	`This projectile's ability to maintain an infrared target lock, which depends on the target ship's heat level and distance from the missile; IR-guided missiles become more accurate the closer they are to their targets.`
 
 tip "radar tracking:"
-	`How well this weapon maintains a radar target lock; negatively affected by the target ship's radar jamming equipment.`
+	`This projectile's ability to maintain a radar target lock; negatively affected by the target ship's radar jamming equipment.`
 
 tip "piercing:"
-	`This percentage of this weapon's damage always leaks through to the hull.`
+	`Percentage of this weapon's damage that leaks through to the hull.`
```
Comment (Amazinite) [inline data/_ui/tooltips.txt:892]:
```suggestion
	`Percentage of this weapon's damage that leaks through to the hull while the target's shields are still active.`
```

File: data/_ui/tooltips.txt [hunk 16]
```diff
@@ -853,109 +853,109 @@ tip "firing burn / second:"
...
 
 tip "piercing resistance:"
-	`What percentage of a weapon's piercing damage is resisted, preventing damage from leaking through the shields to the hull.`
+	`Piercing damage resisted on all incoming projectiles, preventing damage from leaking through the shields to the hull.`
```
Comment (Amazinite) [inline data/_ui/tooltips.txt:895]: This should be up by piercing protection rather than under the weapon tooltips.

File: data/_ui/tooltips.txt [hunk 17]
```diff
@@ -964,67 +964,67 @@ tip "firing fuel / shot:"
...
-	`Each shot fired by this weapon causes this amount of shield loss as a percentage of the ship's total shields.`
+	`Shield damage dealt to your ship per shot, as a percentage of your total shield strength.`
 
 tip "inaccuracy:"
-	`Maximum inaccuracy of this weapon, in degrees.`
+	`Maximum inaccuracy of this weapon, in degrees. A value of one means the weapon's aim will be at most one degree off-target.`
 
 tip "blast radius:"
-	`When this weapon hits something, everything within this distance (including friendly ships) will be damaged.`
+	`Everything within this distance (including friendly ships) will be damaged when the weapon hits something.`
```
Comment (Amazinite) [inline data/_ui/tooltips.txt:1015]: This reminds me: looks like we don't have a tooltip for "safe".

File: data/_ui/tooltips.txt [hunk 18]
```diff
@@ -964,67 +964,67 @@ tip "firing fuel / shot:"
...
 
 tip "missile strength:"
-	`How resistant this missile is to anti-missile shots.`
+	`This missile's resistance to anti-missile shots.`
```
Comment (Amazinite) [inline data/_ui/tooltips.txt:1018]: Not all weapons with missile strength actually fire missiles, so maybe just refer to projectiles in the tooltip.
```suggestion
	`This projectile's resistance to anti-missile shots.`
```

File: data/_ui/tooltips.txt [hunk 19]
```diff
@@ -964,67 +964,67 @@ tip "firing fuel / shot:"
...
 
 tip "anti-missile:"
-	`How powerful this anti-missile system's shots are. Some missiles are harder to destroy than others.`
+	`The power of the anti-missile shots. Higher values make it easier to destroy stronger missiles.`
```
Comment (Amazinite) [inline data/_ui/tooltips.txt:1021]:
```suggestion
	`The power of the anti-missile shots. Higher values make it easier to destroy stronger projectiles.`
```

File: data/_ui/tooltips.txt [hunk 20]
```diff
@@ -964,67 +964,67 @@ tip "firing fuel / shot:"
...
 
 
 
 # Flight checks:
 tip "overheating!"
-	`This ship's outfits are producing so much heat that it will overheat the moment it takes off. You must either add more cooling or remove some of the outfits that are producing heat.`
+	`This ship's outfits are producing so much heat that it will overheat the moment it takes off. You should either add more cooling or remove some of the outfits that are producing heat.`
```
Comment (Amazinite) [inline data/_ui/tooltips.txt:1027]: Anything ending in an exclamation point is something you must resolve in order to actually leave the outfitter, so I think saying "must" instead of "should" for such tooltips makes sense.

File: data/_ui/tooltips.txt [hunk 21]
```diff
@@ -1033,22 +1033,22 @@ tip "no fuel!"
 	`This ship does not have enough fuel to provide energy to power its systems. You should increase the fuel capacity.`
 
 tip "no thruster!"
-	`This ship does not have any sort of thruster installed. If it took off, it would be unable to move.`
+	`This ship is unable to move. You should install some thrusters.`
```
Comment (Amazinite) [inline data/_ui/tooltips.txt:1036]:
```suggestion
	`This ship is unable to move. You should install thrusters.`
```

File: data/_ui/tooltips.txt [hunk 22]
```diff
@@ -1033,22 +1033,22 @@ tip "no fuel!"
...
 
 tip "no steering!"
-	`This ship does not have any steering systems installed. If it took off, it would be unable to turn.`
+	`This ship is unable to turn. You should install some steering systems.`
```
Comment (Amazinite) [inline data/_ui/tooltips.txt:1039]:
```suggestion
	`This ship is unable to turn. You should install steering systems.`
```

File: data/_ui/tooltips.txt (hunk 11 above)
Comment (tibetiroka) [reply data/_ui/tooltips.txt:364]:
```suggestion
	`Protection provided against leak damage. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in leak damage, while a total value of 11 only results in a 10 percent reduction.`
```

File: data/_ui/tooltips.txt (hunk 10 above)
Comment (tibetiroka) [reply data/_ui/tooltips.txt:319]:
```suggestion
	`Protection provided against scrambling damage. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in scramble damage, while a total value of 11 only results in a 10 percent reduction.`
```

File: data/_ui/tooltips.txt (hunk 12 above)
Comment (tibetiroka) [reply data/_ui/tooltips.txt]: Both `the ship` and `this ship` are used quite often.

File: data/_ui/tooltips.txt (hunk 14 above)
Comment (tibetiroka) [reply data/_ui/tooltips.txt]: `Shield damage dealt to the ship per second when firing reverse thrusters.`
Does this look good?

Also, is the accumulative damage for thrust worded properly?

File: data/_ui/tooltips.txt (hunk 17 above)
Comment (tibetiroka) [reply data/_ui/tooltips.txt:1015]: We also don't have tooltips for "phasing", "gravitational" and probably a lot of other weapon attributes. I don't know which ones are displayed in the interface, though.

File: data/_ui/tooltips.txt (hunk 20 above)
Comment (tibetiroka) [reply data/_ui/tooltips.txt:1027]:
```suggestion
	`This ship's outfits are producing so much heat that it will overheat the moment it takes off. You must either add more cooling or remove some of the outfits that are producing heat.`
```

File: data/_ui/tooltips.txt (hunk 13 above)
Comment (tibetiroka) [reply data/_ui/tooltips.txt:436]: Piercing resistance/protection doesn't work with shield disruption, right? So if I say something like "protection provided against damage bypassing shields", that would be misleading.

~~This is hard.~~

```suggestion
	`Protection provided against shield piercing damage. Unlike other protection attributes, this attribute bounces the incoming hull damage to your shields, but doesn't reduce the overall damage dealt. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in damage leak, while a total value of 11 only results in a 10 percent reduction.`
```

I honestly can't think of a better name for piercing "damage".

Comment (Amazinite) [reply data/_ui/tooltips.txt:442]: Feels too wordy. I think the problem is the use of the word damage, because like I said, this isn't actually a damage type.
```suggestion
	`Protection provided against piercing. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in piercing, while a total value of 11 only results in a 10 percent reduction.`
```

File: data/_ui/tooltips.txt (hunk 14 above)
Comment (Amazinite) [reply data/_ui/tooltips.txt]: That looks good. And I believe "accumulated" is used for various thrust costs because those are the DoT damage types. So when you fire a thruster that has 100 ion/sec, you're not losing 100 energy, you're accumulating 100 units of ionization that will decay over the next few seconds. I suppose that brings into question whether "accumulated" should be used for other firing DoT costs, or if the wording of other firing DoT costs should be used for the engine DoT costs.

File: data/_ui/tooltips.txt (hunk 17 above)
Comment (Amazinite) [reply data/_ui/tooltips.txt:1015]: Just realized these are all weapon attributes, which don't appear unless we explicitly allow them to (which is the opposite of outfit attributes which appear by default unless we hide them), and these don't appear to be given tooltips.
